inc/fleet.class.php

```diff
@@ -101,6 +102,8 @@ public function defineTabs($options = []) {
 
    /**
     * Show form for edition
+    * @param $ID
+    * @param array $options
     */
    public function showForm($ID, $options = []) {
```

Comment:
type hint array for $options (becasue you did it earlier i nthis PR)

Reply:
understood